app/models/concerns/triggers/run.rb

```diff
@@ -13,6 +13,24 @@ def run_on run
 
   private
 
+  def line_with_raise backtrace
+    backtrace.find do |line|
+      n = line.match(/\(eval\):(\d+)/)&.captures
+
+      if n.present?
+        # -4 lines of `code` header
```

Comment:
No soy el más _gramático_, pero me parece que se lee mejor:

```suggestion
        # -4 lines of header `code`
```

Reply:
gramaticamente hablando es `la cabecera de code => code header` jaja pero como te guste.